src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int graduationYear;
```

Comment:
The field 'graduationYear' should have an access modifier to encapsulate the field properly. Consider using 'private' to restrict access, unless a wider access scope is intentionally desired.